cmd/main.go

```diff
@@ -534,6 +535,7 @@ func main() {
 
 	go func() {
 		if Options.EnableKubeAPI {
+			failOnError(doesBMHCRDExist(ctrlMgr), "BMH CRD does not exist in cluster")
```

Comment:
Maybe put the whole name here (`BareMetalHost`). Otherwise looks good

Reply:
Good idea, thank you! Modified to reflect full name of CRD.